miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultLibraryService.java

```diff
@@ -242,6 +253,12 @@ public long create(Library library) throws IOException {
       } else {
         transferService.create(transfer);
       }
+      // Don't log initial additions to requisition, as that's part of the requisition creation
+      if (library.getRequisition() != null
+          && library.getRequisition().getCreationTime().toInstant()
+              .isBefore(Instant.now().minus(1, ChronoUnit.HOURS))) {
```

Comment:
Is there a likely case where an addition will be made to a requisition within an hour that isn't the initial one?

Reply:
It's very likely, but that's close enough to consider it part of the creation - the person is most likely creating the requisition in multiple steps